src/common/entity/compute_active_media_states.ts

```diff
@@ -0,0 +1,28 @@
+import type { HassEntity } from "home-assistant-js-websocket";
+import type { HomeAssistant } from "../../types";
+import { generateEntityFilter } from "./entity_filter";
+
+export function computeActiveMediaStates(
+  hass: HomeAssistant,
+  areaId?: string
+): HassEntity[] {
+  const area = areaId ? hass.areas[areaId] : undefined;
+  if (!area) {
+    return [];
```

Comment:
`areaId` is optional, and the function name doesn't imply that this function will only work for areas, and yet if you don't pass an `areaId` the function returns an empty array?

Make the name of the function more descriptive, like get `computeActiveAreaMediaStates`